services/audit-service/src/main/java/datawave/microservice/audit/replay/config/ReplayProperties.java

```diff
@@ -0,0 +1,103 @@
+package datawave.microservice.audit.replay.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+import java.util.concurrent.TimeUnit;
+
+@ConfigurationProperties(prefix = "audit.replay")
+public class ReplayProperties {
```

Comment:
Maybe add validation (using validation annotations) to this? The millis values, I would guess at least all need to be positive, for example.

Reply:
added in next commit.